CONTRIBUTING.md

```diff
@@ -0,0 +1,184 @@
+# Contributing to %PROJECT%
```

Comment:
I think you forgot to replace a value in this document, there are multiple instances of this throughout to take a look at.